NEWS

```diff
@@ -7,6 +7,15 @@ Changes and improvements to testtools_, grouped by release.
 NEXT
 ~~~~
 
+Improvements
+------------
+
+* ``testtools.run`` now accepts ``--locals`` to show local variables
+  in tracebacks, which can be a significant aid in debugging. In doing
+  so we've removed the code reimplementing linecache and traceback as
+  the traceback2 and linecache2 modules can now be used instead.
```

Comment:
I'd call out the new dependency on `traceback2` as a separate bullet point (although, thank heaven that deps aren't quite as much of a pain these days).

Also, nitpick, AIUI, it's not they _can_ be used instead, but rather that we _are_ using them. Is that right?